README.md

```diff
@@ -18,41 +18,73 @@ The script is compatible with any Kerlink Wirnet Station gateway running firmare
 ```
 
 In case gateway was running outdated firmware (e.g. `3.5`), the execution would look like this:
 
 ```
-./provision.sh '192.168.188.101' 'https://thethings.example.com'
+./provision.sh '192.168.4.155' 'https://thethings.example.com'
 CPF is not installed, attempt CPF '1.1.6' installation?[y/n]y
 Gateway firmware version is '3.5', which is lower than '3.6' required by CPF, attempt firmware update to '3.6'?[y/n]y
 Downloading firmware from 'https://github.com/TheThingsNetwork/kerlink-station-firmware/releases/download/wirnet-3.6/fwupgrade_wirmav2_wirnet_v3.6.tar.gz' to '/run/user/1000/tmp.3sb4TlkWRc/dota.tar.gz' locally...
 Pushing '/run/user/1000/tmp.3sb4TlkWRc/dota.tar.gz' to '/mnt/fsuser-1/dota/dota.tar.gz'...
 Firmware successfully pushed, reboot gateway to start update?[y/n]y
-Waiting for 192.168.188.101 to reboot............[down]......................................
-# Host 192.168.188.101 found: line 30
+Waiting for 192.168.4.155 to reboot............[down].....................................
+# Host 192.168.4.155 found: line 30
 /home/rvolosatovs/.ssh/known_hosts updated.
 Original contents retained as /home/rvolosatovs/.ssh/known_hosts.old
+The authenticity of host '192.168.4.155 (192.168.4.155)' can't be established.
+ECDSA key fingerprint is SHA256:yr/giQOwLcT8+ZgDrDKg/ca4XMdOfiqSivrPZBW3d9o.
+Are you sure you want to continue connecting (yes/no)? yes
+Warning: Permanently added '192.168.4.155' (ECDSA) to the list of known hosts.
+root@192.168.4.155's password: 
 Downloading firmware from 'https://github.com/TheThingsNetwork/kerlink-station-firmware/releases/download/wirnet-3.6/custo_knetd-4.12.tar.gz' to '/run/user/1000/tmp.SVnxRHfdfD/dota.tar.gz' locally...
 Pushing '/run/user/1000/tmp.SVnxRHfdfD/dota.tar.gz' to '/mnt/fsuser-1/dota/dota.tar.gz'...
+root@192.168.4.155's password: 
 Firmware successfully pushed, reboot gateway to start update?[y/n]y
-Waiting for 192.168.188.101 to reboot............[down]............
+Waiting for 192.168.4.155 to reboot............[down]............
 Downloading firmware from 'https://github.com/TheThingsNetwork/kerlink-station-firmware/releases/download/cpf-1.1.6/dota_cpf_1.1.6-1.tar.gz' to '/run/user/1000/tmp.Qr60nW4dWc/dota.tar.gz' locally...
 Pushing '/run/user/1000/tmp.Qr60nW4dWc/dota.tar.gz' to '/mnt/fsuser-1/dota/dota.tar.gz'...
 Firmware successfully pushed, reboot gateway to start update?[y/n]y
-Waiting for 192.168.188.101 to reboot............[down]....................................
+Waiting for 192.168.4.155 to reboot...........[down]...................................
+Querying available frequency plan templates...
+Available frequency plan templates are:
+0) AS923-JP_8CH
+1) AU915-AU_8CH
+2) EU868-FR_8CH
+3) IN865-IN_8CH
+4) RU864-RU_8CH
+5) RU864-RU_8CH_Legacy
+6) US915-US_8CH
+Please select a frequency plan template to use for lorad configuration.[0-6]2
+
+Setting frequency plan template to 'EU868-FR_8CH'...
 Setting LNS address to https://thethings.example.com...
```

Comment:
Is this really https? This is UDP right?

Reply:
I think it should be udp - I just used the same address as used in TTKG documentation - https://github.com/TheThingsNetwork/lorawan-stack/commit/341992791831688f7ee181672530f6aeba95f45c#diff-7a24307be2f33b16301f7eb811aabacaR39 (cc @htdvisser)

What should this be then? `udp://thethings.example.com` ? I guess just `thethings.example.com` is enough, right?

Reply:
Yes, just the host name